tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -211,6 +211,30 @@ async def eventual_balance():
             transactions = await client.get_transactions("1")
             assert len(transactions) > 1
 
+            # TODO: make this "get all transactions" feature test be not garbage.
+            #       for example, taking 14 seconds is a bit attrocious.
```

Comment:
This really seems like a mess that I've coded here.  I trust it will inspire someone to educate me on a decent way to create a pile of transactions to test getting many transactions through the RPC.

Reply:
Maybe there's not a great way, but I've gotten something better at least.